src/components/Loader/Loader.tsx

```diff
@@ -13,7 +13,7 @@ const Loader = () => (
       <div className={styles.back}>
         <div className={styles.earth}></div>
         <div className={styles.cosmos}>
-          <div className={styles.star}></div>
+          <div className={styles.star} data-happo-hide></div>
```

Comment:
Q: Should this impact the happo diffs on this PR, or in future PRs?

Reply:
I think we should ignore the loader in Happo. It has very tightly scoped CSS and it is unlikely it would be affected by other styling inadvertently. So hopefully, this gets it to stop showing up. It is getting caught every time because the stars in the background generate randomly so its capturing a new image every run 🤦‍♀️

Reply:
I expect happo to have a diff on the loader this time, one with a star field from before and mine without. But in future PRs the star field won't display for happo at all so shouldn't cause a diff.